ignite/cmd/plugin.go

```diff
@@ -18,33 +18,46 @@ import (
 	"github.com/ignite/cli/ignite/services/plugin"
 )
 
-// plugins hold the list of plugin declared in the config.
-// A global variable is used so the list is accessible to the plugin commands.
-var plugins []*plugin.Plugin
-
 const (
 	igniteCmdPrefix = "ignite "
 )
 
+// plugins hold the list of plugin declared in the config.
+// A global variable is used so the list is accessible to the plugin commands.
+var plugins []*plugin.Plugin
+
 // LoadPlugins tries to load all the plugins found in configuration.
 // If no configuration found, it returns w/o error.
 func LoadPlugins(ctx context.Context, rootCmd *cobra.Command) error {
-	cfg, _, err := parseLocalPlugins(rootCmd)
-	if err != nil {
-		// if binary is run where there is no plugins.yml, don't load
-		return nil
+	localCfg, _, err := parseLocalConfig(rootCmd)
+	// if binary is run where there is no cfg, don't load
+	if err == nil {
```

Comment:
What error we are preventing here? Maybe we should throw the exact error from the child method and handle it here eg:

```suggestion
	if err == ErrNotConfig {
```

wdyt?

Reply:
Yeah, I think actually in the case where it is `ErrConfigNotFound` the function should continue.

Reply:
Refactored now.